streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -366,6 +366,26 @@ public void testShouldTransitToNotRunningIfCloseRightAfterCreated() {
         }
     }
 
+    @Test
```

Comment:
Shouldn't you also verify if the tasks are initialized when KafkaStreams is started?

Reply:
14cf866